packages/flutter/test/material/search_anchor_test.dart

```diff
@@ -1769,6 +1769,50 @@ void main() {
     expect(searchViewRect.topLeft, equals(const Offset(rootSpacing, rootSpacing)));
   });
 
+  testWidgets('Docked search should with nested navigator does not go off the screen', (WidgetTester tester) async {
```

Comment:
nit: Is "should" an extra word?
```suggestion
  testWidgets('Docked search view with nested navigator does not go off the screen', (WidgetTester tester) async {
```